doc/POINTS_COORDINATES.md

```diff
@@ -0,0 +1,169 @@
+For example:
+
+```c++
+point_abs_ms pos_ms = get_avatar()->global_square_location().xy();
+point_abs_omt pos_omt = project_to<coords::omt>( pos_ms );
+assert( pos_omt == get_avatar()->global_omt_location().xy() );
+```
+
+The same function `project_to` can be used for scaling up or down.  When
+converting to a coarser coordinate system precision is of course lost.  If you
+care about the remainder then you can instead use `project_remain`.
+
+`project_remain` allows you to convert to a coarser coordinate system and also
+capture the remainder relative to that coarser point.  It returns a helper
+struct intended to be used with `std::tie` to capture the two parts of the
+result.  For example, suppose you want to know which overmap the avatar is in,
+and which overmap terrain they are in within that overmap.
+
+```c++
+point_abs_omt abs_pos = get_avatar()->global_omt_location().xy();
+point_abs_om overmap;
+point_om_omt omt_within_overmap;
+std::tie( overmap, omt_within_overmap ) = project_remain<coords::om>( abs_pos );
+```
+
+That makes sense for two-dimensional `point` types, but how does it handle
+`tripoint`?  Recall that the z-coordinates do not scale along with the
+horizontal dimensions, so `z` values are unchanged by `project_to` and
+`project_remain`.  However, for `project_remain` we don't want to duplicate the
+z-coordinate in both parts of the result, so you must choose exactly one to be
+a `tripoint`.  In the example above, z-coodinates do not have much meaning at
+the overmap scale, so you probably want the z-coordinate in
+`omt_within_overmap`.  Than can be done as follows:
+
+```c++
+tripoint_abs_omt abs_pos = get_avatar()->global_omt_location();
+point_abs_om overmap;
+tripoint_om_omt omt_within_overmap;
+std::tie( overmap, omt_within_overmap ) = project_remain<coords::om>( abs_pos );
+```
+
+The last available operation for rescaling points is `project_combine`.  This
+performs the opposite operation from `project_remain`.  Given two points where
+the origin of the second matches the scale of the first, you can combine them
+into a single value.  As you might expect from the above discussion, one of
+these two can be a `tripoint`, but not both.
+
+```c++
+tripoint_abs_omt abs_pos = get_avatar()->global_omt_location();
+point_abs_om overmap;
+tripoint_om_omt omt_within_overmap;
+std::tie( overmap, omt_within_overmap ) = project_remain<coords::om>( abs_pos );
+tripoint_abs_omt abs_pos_again = project_combine( overmap, omt_within_overmap );
+assert( abs_pos == abs_pos_again );
+```
+
+### Changing origin
+
+`project_remain` and `project_combine` facilitate some changes of origin, but
+only those origins specifically related to rescaling.  To convert to or from
+local or vehicle coordinates requires a specific `map` or `vehicle` object.
+
+TODO: write some examples once this is implemented.
+
+## Point operations
+
+The standard arithmetic operations you might expect should be available for all
```

Comment:
I would expect all arithmetic operations to be available, but they aren't in many circumstances, so this sentence is very confusing to me.

Reply:
Rephrased.